test/conformance/api/v1/revision_timeout_test.go

```diff
@@ -132,9 +185,9 @@ func TestRevisionTimeout(t *testing.T) {
 				t.Fatalf("Error probing %s: %v", serviceURL, err)
 			}
 
-			if err := sendRequest(t, clients, serviceURL, tc.initialSleep, tc.sleep, tc.expectedStatus); err != nil {
-				t.Errorf("Failed request with initialSleep %v, sleep %v, with revision timeout %ds and expecting status %v: %v",
-					tc.initialSleep, tc.sleep, tc.timeoutSeconds, tc.expectedStatus, err)
+			if err := sendRequest(t, clients, serviceURL, tc.initialSleep, tc.sleep, tc.expectedStatus, tc.expectedBody); err != nil {
+				t.Errorf("Failed request with initialSleep %v, sleep %v, with revision timeout %ds,expecting status %v and expecting body %q: %v",
```

Comment:
```suggestion
				t.Errorf("Failed request with initialSleep %v, sleep %v, with revision timeout %ds, expecting status %v and body %q: %v",
```

Reply:
> Looks like this is resolved no?
Which one?

Reply:
GitHub's threading isn't great - was looking at
https://github.com/knative/serving/pull/12322/files?file-filters[]=.go#r754285374

Reply:
Yes true :) I added the test by extending the conformance ones related to timeouts.